sprout/appliances/tasks.py

```diff
@@ -311,14 +311,15 @@ def poke_trackerbot(self):
                 with transaction.atomic():
                     tpl = Template(
                         provider=provider, template_group=group, original_name=template_name,
-                        name=template_name, preconfigured=False, date=date, custom_data=custom_data,
+                        name=template_name, preconfigured=False, date=date,
+                        custom_data=processed_custom_data,
                         version=template_version, ready=True, exists=True, usable=True)
                     tpl.save()
                     if provider.provider_type == 'openshift':
                         tpl.custom_data = yaml.safe_load(processed_custom_data)
```

Comment:
**QUESTION**: Here is yaml load, but on line 315 it is set directly. I don't understand it, it can digest it like this?

Reply:
Tagging as **REQUIRED**

Reply:
@mfalesni , @psav, optimized yaml calls.